### PR TITLE
Updating extension for ppom-validator updates

### PR DESCRIPTION
**Pull request by jpuri**
Changes in extension for new changes in ppom-validator package.

### CODE REVIEW

**Comment by HowardBraham** [comment]
Congrats, you hit PR #20000!!

**Comment by jpuri** [comment]
@SocketSecurity ignore-all

**Comment by jpuri** [comment]
@metamaskbot update-policies

**Comment by jpuri** [comment]
@metamaskbot update-policies

**Comment by jpuri** [comment]
@metamaskbot update-policies

File: app/scripts/lib/ppom/ppom-middleware.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { PPOM } from '@blockaid/ppom';
+
+import { PPOMController } from '@metamask/ppom-validator';
+
+const ConfirmationMethods = [
```
Comment (cryptotavares) [inline app/scripts/lib/ppom/ppom-middleware.ts:5]: Should we freeze this array? `Object.freeze([...])` (just to be sure that no dep is changing the whitelisted methods)

File: app/scripts/lib/ppom/ppom-middleware.test.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  writable: true,
+  value: () => undefined,
+});
+
+Object.defineProperty(globalThis, 'performance', {
+  writable: true,
+  value: () => undefined,
+});
+
+describe('PPOMMiddleware', () => {
```
Comment (cryptotavares) [inline app/scripts/lib/ppom/ppom-middleware.test.ts:13]: Should we also add a test with a method that is not whitelisted for ppom (method not in ConfirmationMethods) and ensure that usePPOM is not executed?

**Review by cryptotavares — DISMISSED** [review]
Left two really minor comments, but looks good 👍

**Comment by jpuri** [comment]
@cryptotavares : I have addressed the feedbacks, can you plz re-approve.

**Review by cryptotavares — APPROVED** [review]

**Review by segun — APPROVED** [review]

**Pull request merged**